js/data/index.d.ts

```diff
@@ -42,7 +42,14 @@ export type SelectDescriptor<T> = string | Array<string> | ((source: T) => any);
  * @public
  */
 export type FilterDescriptor = any;
-
+/**
+ * @docid
+ * @public
+ */
+export type LangParams = {
+  locale: string;
+  collateOptions?: Intl.CollatorOptions;
```

Comment:
```suggestion
  collatorOptions?: Intl.CollatorOptions;
```
I thought we agreed with PMs that 'collator' is better than 'collate'

Reply:
ok, i fixed it